src/proteomes/components/entry/ComponentsDownload.tsx

```diff
@@ -260,14 +232,29 @@ const ComponentsDownload = ({
           checked={downloadSelect === 'all'}
           onChange={handleDownloadAllChange}
         />
-        Download all{' '}
-        {showReviewedOption
-          ? 'reviewed (Swiss-Prot) and unreviewed (TrEMBL) proteins'
-          : ''}{' '}
-        {/* (<LongNumber>{showReviewedOption ? isoformStats?.allWithIsoforms : totalNumberResults}</LongNumber>) */}
-        (<LongNumber>{totalNumberResults}</LongNumber>{' '}
-        {showReviewedOption ? ' + isoforms' : ''})
+        Download all reviewed (Swiss-Prot) and unreviewed (TrEMBL) proteins (
+        <LongNumber>
+          {includeIsoform
+            ? totalNumberResults + (statistics.isoforms || 0)
+            : totalNumberResults}
+        </LongNumber>
+        )
       </label>
+      {isoformsAvailable && (
+        <div className={styles['isoform-option']}>
+          Additional sequence data
+          <label htmlFor="data-selection-isoform">
+            <input
+              id="data-selection-isoform"
+              type="checkbox"
+              name="reviewed-isoform"
+              onChange={handleIsoformSelect}
+            />
+            Include reviewed (Swiss-Prot) isoforms --{' '}
```

Comment:
```suggestion
            Include reviewed (Swiss-Prot) isoforms ―{' '}
```
Suggestion to use a unicode horizontal bar instead of two dashes, to avoid seeing the bit of space between the two dashes but still have a long separation